llvm/lib/Target/RISCV/RISCVSchedSpacemitX60.td

```diff
@@ -232,9 +392,660 @@ let Latency = 4 in {
+      defm "" : LMULWriteResMX<"WriteVSTUX" # eew, [SMX60_VLS], mx, IsWorstCase>;
+      defm "" : LMULWriteResMX<"WriteVSTOX" # eew, [SMX60_VLS], mx, IsWorstCase>;
+    }
+  }
+}
+
+// Segmented loads and stores: base latency multiplied by number of fields
+// TODO: These latencies are estimations and are not confirmed experimentally
+foreach mx = SchedMxList in {
+  foreach nf=2-8 in {
+    foreach eew = [8, 16, 32, 64] in {
+      defvar IsWorstCase = SMX60IsWorstCaseMX<mx, SchedMxList>.c;
+      defvar EEWMultiplier = !div(eew, 8);
+      defvar StridedLatency = !mul(Get4458Latency<mx>.c, EEWMultiplier);
+
+      // Unit-stride segmented
+      let Latency = !mul(Get44816Latency<mx>.c, nf) in {
+        defm "" : LMULWriteResMX<"WriteVLSEG" # nf # "e" #eew, [SMX60_VLS], mx, IsWorstCase>;
+        defm "" : LMULWriteResMX<"WriteVLSEGFF" # nf # "e" #eew, [SMX60_VLS], mx, IsWorstCase>;
+        defm "" : LMULWriteResMX<"WriteVSSEG" # nf # "e" #eew, [SMX60_VLS], mx, IsWorstCase>;
+      }
+
+      // Strided/indexed segmented
+      let Latency = !mul(StridedLatency, nf) in {
+        defm "" : LMULWriteResMX<"WriteVLSSEG" # nf # "e" #eew, [SMX60_VLS], mx, IsWorstCase>;
+        defm "" : LMULWriteResMX<"WriteVLUXSEG" # nf # "e" #eew, [SMX60_VLS], mx, IsWorstCase>;
+        defm "" : LMULWriteResMX<"WriteVLOXSEG" # nf # "e" #eew, [SMX60_VLS], mx, IsWorstCase>;
+
+        defm "" : LMULWriteResMX<"WriteVSSSEG" # nf # "e" #eew, [SMX60_VLS], mx, IsWorstCase>;
+        defm "" : LMULWriteResMX<"WriteVSUXSEG" # nf # "e" #eew, [SMX60_VLS], mx, IsWorstCase>;
+        defm "" : LMULWriteResMX<"WriteVSOXSEG" # nf # "e" #eew, [SMX60_VLS], mx, IsWorstCase>;
+      }
+    }
+  }
+}
+
+// Whole register move/load/store
+// TODO: These latencies are estimations and are not confirmed experimentally
+foreach LMul = [1, 2, 4, 8] in {
+  let Latency = Get461018Latency<!strconcat("M", !cast<string>(LMul))>.c in {
+    def : WriteRes<!cast<SchedWrite>("WriteVLD" # LMul # "R"), [SMX60_VLS]>;
+    def : WriteRes<!cast<SchedWrite>("WriteVST" # LMul # "R"), [SMX60_VLS]>;
+  }
+}
+
+let Latency = 4 in {
+  def : WriteRes<WriteVMovSX, [SMX60_VIEU]>;
+  def : WriteRes<WriteVMovFS, [SMX60_VIEU]>;
+  def : WriteRes<WriteVMovSF, [SMX60_VIEU]>;
+  def : WriteRes<WriteVMov1V, [SMX60_VIEU]>;
+  def : WriteRes<WriteVMov2V, [SMX60_VIEU]>;
+}
+def : WriteRes<WriteVMovXS, [SMX60_VIEU]> { let Latency = 6; }
+def : WriteRes<WriteVMov4V, [SMX60_VIEU]> { let Latency = 8; }
+def : WriteRes<WriteVMov8V, [SMX60_VIEU]> { let Latency = 16; }
+
+// 11. Vector Integer Arithmetic Instructions
+foreach mx = SchedMxList in {
+  defvar IsWorstCase = SMX60IsWorstCaseMX<mx, SchedMxList>.c;
+
+  let Latency = Get4458Latency<mx>.c, ReleaseAtCycles = [Get1248Latency<mx>.c] in {
+    defm "" : LMULWriteResMX<"WriteVIMinMaxV", [SMX60_VIEU], mx, IsWorstCase>;
+    defm "" : LMULWriteResMX<"WriteVIMinMaxX", [SMX60_VIEU], mx, IsWorstCase>;
+  }
+
+  let Latency = Get44816Latency<mx>.c in {
```

Comment:
Some of these have ReleaseAtCycle, and some don't, but I don't see any obvious pattern?  Am I missing something here?